Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM python:3.8.2-buster as base
+
+RUN pip install gunicorn
+RUN pip install poetry
```

Comment:
Poetry should be installed via the recommended method:
![image](https://user-images.githubusercontent.com/8018173/100366420-b5988a80-2ff8-11eb-94d9-c62678ef2b35.png)